### ISSUE TITLE
Error in Diablo.js.. diablo not found

### ISSUE THREAD

**Issue by silentdark6** [kolton/d2bot-with-kolbot#978]
Error in Diablo.js.. diablo not found

is there anyone way make the sorc kill light immunes or atleast stand there and try so my merc can go kill it?.. iv tried everything from 

tele stomp(which makes him tele stomp EVERY light immune, which i dont need to since i have infinity) 
to 
adding the skip exception to seis viz and lord to make sure she kills them
to 
adding time delay to wait for d at the star..
to
setting it to kill ALL monster in a given area
to
setting lighting "my main skill" to Config.AttackSkill[5] = 49; // Secondary skill if monster is immune to primary.

iv looked through the diablo.js to see if there was anything else in there to make sure she kills viz but i didnt see anything.. does anyone have a solution for this??  Mind you it doesnt happen every single time but i would say atleast 40-50% of the time she misses d because she wont kill viz..
she will kill every other light immune due to infinity but i guess the mini bosses have a lot more light immunity then reg ones.. also using a -25%/20 griffons eye so light immunes are no problem dont know why she skips him..

please help,
thanks in advance :D

**Comment by janez12** [comment]
a simple solution is to put one skill point in ice bolt and set that to skill if immune. other than that i have the same problem

**Comment by silentdark6** [comment]
but then i wont do 50k dmg :( bahaha  that is the only thing i have not tried.. i was waiting till i hit 96(95atm) to start putting them into fire so atleast she should attack the mod and merc  can kill it :P

**Comment by janez12** [comment]
dont go fire. he can be light and fire immune but he will never be light and cold immune. and i know it takes time but with a solid dmg infinity it shouldnt take more than a few seconds.

**Comment by 5noop** [comment]
I don't see any clean solution without editing attack.js, but you could still try one of these things:
- Use normal attack (1) against lightning immune
- Put a delay in Diablo.js to make the sorc stand still if she fails to kill Seis, ideally that would then give enough time for the merc to kill him. So in the vizierSeal function around line 210 you would have this:
```
if (!this.getBoss(getLocaleString(2851))) {
	delay(10000); // 10 sec delay
}
```
@janez12 Seis can actually be cold + lightning immune, that's a thing I notice playing blizz/static in classic

**Comment by janez12** [comment]
yes i think seis is actually always cold immune but i think the question was about vizier.

**Comment by 5noop** [comment]
Yeah my bad i was thinking about vizier too
